diff_drive_controller/msg/DiffDriveControllerState.msg

```diff
@@ -0,0 +1,28 @@
+# This initial part is the same as JointTrajectoryControllerState.msg:
```

Comment:
Looks great— will be awesome to have for diagnostic purposes. Thanks for doing this.